coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/StaticHiveFunctionRegistry.java

```diff
@@ -267,6 +267,7 @@ public boolean isOptional(int i) {
     createAddUserDefinedFunction("unix_timestamp", BIGINT,
         family(ImmutableList.of(SqlTypeFamily.STRING, SqlTypeFamily.STRING), optionalOrd(ImmutableList.of(0, 1))));
     createAddUserDefinedFunction("to_date", HiveReturnTypes.STRING, or(STRING, DATETIME));
+    createAddUserDefinedFunction("date", HiveReturnTypes.STRING, or(STRING, DATETIME));
```

Comment:
@ljfgem If I understand correctly, `date` function should return DATE type but it seems returning STRING type here.